python/subunit/tests/test_test_protocol2.py

```diff
@@ -17,6 +17,10 @@
 from io import BytesIO
 import datetime
 
+from hypothesis import given
+# from hypothesis import Settings, Verbosity
+# Settings.default.verbosity = Verbosity.verbose
```

Comment:
So there's no need for it, but its a useful hint for this new thing I think. Do you object to it?